### DISCUSSION TITLE
Error with Illustrating Electrostatic Repulsion

### DISCUSSION THREAD

**Discussion by hookeba** [dipy/dipy#2623] [Preprocessing]
## Description

Hello DIPY experts!

While exploring the tutorial “Gradients and Spheres”, I noticed that the illustration of electrostatic repulsion does not appear in a new window although the image <initial_vs_updated.png> is saved in my home directory.  Your expert assistance with troubleshooting is greatly appreciated!

Respectfully yours

hookeba

- [x] Operating system and version (run `python -c "import platform; print(platform.platform())"`)
 Linux-3.10.0-1160.66.1.el7.x86_64-x86_64-with-glibc2.17
- [x] Python version (run `python -c "import sys; print("Python", sys.version)"`)
Python 3.10.5
- [x] dipy version (run `python -c "import dipy; print(dipy.__version__)"`)
 1.5.0
- [x] dependency version (numpy, scipy, nibabel, h5py, cvxpy, fury)
    * import numpy; print("NumPy", numpy.__version__)
    * NumPy 1.23.1
    * import scipy; print("SciPy", scipy.__version__)
    * SciPy 1.8.1
    * import nibabel; print("Nibabel", nibabel.__version__)
    * Nibabel 4.0.1
    * import h5py; print("H5py", h5py.__version__)
    * H5py 3.7.0
    * import cvxpy; print("Cvxpy", cvxpy.__version__)
    * Cvxpy 1.2.1
    * import fury; print("fury", fury.__version__)
    * fury 0.8.0
[gradients_spheres.zip](https://github.com/dipy/dipy/files/9114056/gradients_spheres.zip
![initial_vs_updated](https://user-images.githubusercontent.com/50643872/179058102-94448447-bf6a-40d4-b041-a544ae69882f.png)
)

**Comment by skoudoro** [comment]
Hi @hookeba,

You need to put the variable `interactive` to `True` instead of `False` if I understand well your issue. 

Please, let me know if I misunderstood it. 

I will convert this issue to discussion since it is not an issue on the codebase.

**Comment by hookeba** [comment]
Hello @skoudoro 

Thank you for your kind response.  I forgot to mention that I also experimented by setting <interactive = True>, but the window still failed to open.  In a remote workflow, I am aware of display issues with MATLAB as rendering is performed on the local desktop.  Are there any comparable issues with DIPY?

Respectfully yours

@hookeba
